pkg/collector/python/datadog_agent.go

```diff
@@ -248,6 +248,11 @@ type sqlConfig struct {
 	DollarQuotedFunc bool `json:"dollar_quoted_func"`
 	// ReturnJSONMetadata specifies whether the stub will return metadata as JSON.
 	ReturnJSONMetadata bool `json:"return_json_metadata"`
+
+	// ObfuscationMode specifies the obfuscation mode to use for go-sqllexer pkg.
+	// When specified, obfuscator will attempt to use go-sqllexer pkg to obfuscate (and normalize) SQL queries.
+	// Valid values are "obfuscate_only", "obfuscate_and_normalize"
+	ObfuscationMode string `json:"obfuscation_mode" yaml:"obfuscation_mode"`
```

Comment:
Sorry to keep piling on the topic but I think it might be nicer if we used the type alias here (you will have to make it an exported type to use it here).

Reply:
I thought about that but given this is primarily for python binding, I feel it makes sense to use a native string type.

Reply:
It's a good thing to think about. I'm not sure how the type aliasing works there. I thought maybe the fact that it's really a string behind the alias would make this transparent but I've never validated that assumption.

Reply:
I will give it a try and do some test from python code to make sure it's working

Reply:
type alias works fine with json.unmarshall.